Kickstarter-iOS/Views/Cells/PledgePaymentMethodsCell.swift

```diff
@@ -0,0 +1,84 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class PledgePaymentMethodsCell: UITableViewCell, ValueCell {
+
+  private let cellIdentifier = "Cell"
```

Comment:
You can remove this and just use the cell's `defaultReusableId` property which is swizzled for us.

Reply:
Or even better you should dequeue the cell based on its class name like we do for `UITableViewCell`s (might need some additional `UICollectionView` support)

Reply:
I would go with dequeueing the cell based on its class name, but since this PR doesn't include the creation of the CreditCardCell class, for now I will use `defaultReusableId`. Does it sound good?